src/App.js

```diff
@@ -35,6 +36,8 @@ class App extends Component {
             component={ViewAnalytics}
           />{' '}
           {/* View Analytics Redirect */}
+          <Route exact path="/Draft" component={Draft} />{' '}
```

Comment:
this will need to be behind an `AuthenticatedRoute` once the auth changes are merged in